src/locales/cn.json

```diff
@@ -415,10 +412,7 @@
 
     "intro": [{
       "header": "欢迎来到 æternity 命名系统(AENS)!",
-      "body": "Base æpp钱包的这部分目前为测试版. 我们正在积极改进AENS的功能，很兴奋能与您分享这个过程."
-    }, {
-      "header": "AENS 如何工作?",
-      "body": "AENS允许用户申请一个名称,并与一个或多个æternity帐户关联. 目前为止,只有以.test结尾的名字可以申请.您成功申请后,它会与您的帐户关联. At that point anyone can send tokens by typing the name instead of the account address."
+      "body": "AENS允许用户申请一个名称,并与一个或多个æternity帐户关联. 您成功申请后,它会与您的帐户关联. At that point anyone can send tokens by typing the name instead of the account address."
```

Comment:
```suggestion
      "body": "AENS允许用户申请一个名称,并与一个或多个æternity帐户关联. 您成功申请后、它会与您的帐户关联. At that point anyone can send tokens by typing the name instead of the account address."
```